src/components/Spacing/Spacing.tsx

```diff
@@ -13,22 +13,34 @@
  * limitations under the License.
  */
 
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
 import { css } from '@emotion/core';
+import styled, { StyleProps } from '../../styles/styled';
+
+export interface SpacingProps {
+  /**
+   * Adds bottom margin to component
+   */
+  bottom?: boolean;
+
+  /**
+   * Adds top margin to component
+   */
+
+  top?: boolean;
```

Comment:
These empty lines might break the [JSDoc](https://jsdoc.app/) annotations, please remove them.